forms/gridfield/GridFieldDetailForm.php

```diff
@@ -752,6 +752,21 @@ public function getTemplate() {
 	}
 
 	/**
+	 * Get list of templates to use
+	 *
+	 * @return array
+	 */
+	public function getTemplates()
+	{
+		$templates = SSViewer::get_templates_by_class($this, '', __CLASS__);
```

Comment:
Yeah, I've started using this extensively to save having to use literal class strings (which are undesirable due to their difficulty to detect and upgrade).